Jenkinsfile

```diff
@@ -189,10 +189,16 @@ pipeline {
                     unstash 'mac_installer_test'
                     unstash 'win_installer_test'
 				}
+				script {
+					if (params.UPDATE_DICTIONARIES) {
```

Comment:
Do we still want to only conditionally update dicts?

Reply:
Well, to be honest, I could not find anyone to answer that question. I left it as it was under the assumption that there is some reason behind it.